singular.js

```diff
@@ -4,10 +4,10 @@ module.exports = function (app, bundle) {
   //
   // Singular bundles
   //
-  app.get('/bundle/:module', singular(bundle));
-  app.get('/debug-bundle/:module', singular(bundle, { debug: true }));
-  app.get('/standalone/:module', singular(bundle, { standalone: true }));
-  app.get('/debug-standalone/:module', singular(bundle, { standalone: true, debug: true }));
+  app.get('/bundle/:scope?/:module', singular(bundle));
```

Comment:
Oh neat, I've actually never written a collapsing parameter like that.

But maybe we should go for @scope%2Fmodule ?